frontend/src/views/help/Help.tsx

```diff
@@ -15,7 +15,7 @@ function Help() {
   return (
     <ScreenLayout image={metas.background}>
       <div className="text-center h-screen flex flex-col justify-center items-center select-none gap-11">
-        <BackButton url="/home" imageSrc="images/backbutton.png" />
+        <BackButton url="/home" />
         <h1 className="text-8xl mt-24  mb-10 max-w-[300px]">
           {metas.textCenter.title}
         </h1>
```

Comment:
valida si "title" tiene valor, si no tiene pon uno por default:

```suggestion
        <h1 className="text-8xl mt-24  mb-10 max-w-[300px]">
          {metas?.textCenter?.title ?? ''}
        </h1>
```